packages/language-server/src/ui5-model.ts

```diff
@@ -173,13 +155,14 @@ export async function getSemanticModelWithFetcher(
     })
   );
 
-  return generate({
+  const model = generate({
     version: version,
     libraries: jsonMap,
     typeNameFix: getTypeNameFix(),
     strict: false,
     printValidationErrors: false,
   });
+  return (semanticModelCache[key] = model);
```

Comment:
this is subjective but I am not a fan of performing two actions:
- store cache
- return value

in the same statement, but maybe this is idiomatic style when dealing with caches?

Reply:
If you prefer, I change this